dragonfly/implementations/__init__.py

```diff
@@ -28,3 +28,9 @@
 # keep these out of sequence, they inherit from elsewhere in dragonfly
 from .lockin_provider import *
 from .muxer_provider import *
+
+#from .sql_snapshot import *
```

Comment:
do these imports need to be out of sequence for some reason? If so please describe in a comment line, otherwise please alphabetize them with the imports above.